frontend/components/dashboard/chart/RewardsChart.vue

```diff
@@ -205,6 +207,7 @@ const option = computed<ECBasicOption | undefined>(() => {
       silent: true,
       axisLabel: {
         formatter: valueFormatter.value,
+        fontWeight: fontWeightMedium,
         fontSize: textSize,
```

Comment:
We have 3 occurences of `fontSize: textSize`
Would it be safer to write `fontSize: textSize + 'px'` ?

Reply:
Fontsize in the ECharts is a number ... so adding 'px' would not be valid
https://echarts.apache.org/en/option.html#textStyle.fontSize